src/uu/du/src/du.rs

```diff
@@ -80,6 +86,12 @@ Otherwise, units default to 1024 bytes (or 512 if POSIXLY_CORRECT is set).
 SIZE is an integer and optional unit (example: 10M is 10*1024*1024).
 Units are K, M, G, T, P, E, Z, Y (powers of 1024) or KB, MB,... (powers
 of 1000).
+
+PATTERN is based on the globset crate:
```

Comment:
This might be more confusing that helpful for users. The patterns should probably be described as normal globs instead.